src/omnicore/omnicore.cpp

```diff
@@ -3803,6 +3803,9 @@ int mastercore_handler_block_end(int nBlockNow, CBlockIndex const * pBlockIndex,
         PrintToLog("Consensus hash for block %d: %s\n", nBlockNow, consensusHash.GetHex());
     }
 
+    // request utdb sanity check
+    p_utdb->SanityCheck();
```

Comment:
Hm.. is there a specific reason why we need this? Not that I don't think it's a bad idea, but it potentially slows things a bit down.

Reply:
It does have a performance hit yep, it's actually not too bad but it does add a little overhead.  The only reason I have it in is to try and catch any errors...